### PR TITLE
fix(breadcrumbs): hide the last breadcrumb if it’s not a link

### LINKED ISSUE #6419: Breadcrumbs should not display the title of the current page
On the discussion pages I noticed that the title shows up no less than three times right next to each other. The breadcrumbs instance of the title is almost certainly superfluous because its not even clickable.

### PR DESCRIPTION
**Pull request by mrclay**
Fixes #6419 (breaks BC)

### CODE REVIEW

**Comment by ewinslow** [comment]
We have a feature in the channeling generator where you can explicitly call out a breaking change in the commit message and it will be included in its own section in the change log. Can you do that, or should we just settle for updating the docs?

File: docs/guides/upgrading.rst [hunk 1]
```diff
@@ -3,12 +3,22 @@ Upgrading Plugins
...
 .. contents:: Contents
    :local:
    :depth: 2
 
+From 1.11 to 2.0
+================
+
+Breadcrumb display now removes the last item if it does not contain a link. If you need the previous behavior,
+unregister the handler ``elgg_prepare_breadcrumbs`` for the plugin hook ``'prepare', 'breadcrumbs'`` and replace
+it with a custom handler that only applies the excerpt.
```
Comment (ewinslow) [inline docs/guides/upgrading.rst]: "A custom handler that only applied the excerpt" is not super clear to me. Can you also include a code example?

**Comment by ewinslow** [comment]
Thank you for starting the work on master!

**Comment by mrclay** [comment]
I improved the docs. How do you markup the commit log?

**Comment by beck24** [comment]
LGTM

**Pull request merged**